cms/util.py

```diff
@@ -0,0 +1,18 @@
+"""
+Utility functions for CMS models
+"""
+from urllib.parse import urlparse, parse_qs
+
+
+def get_coupon_code(request):
+    coupon_code = request.GET.get("coupon", None)
```

Comment:
Can we skip this case and just look in `next`? We should only have one URL for a coupon link